st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -276,7 +276,7 @@ def validate(self):
             return self
 
     @jsexpose(body_cls=ExecutionSpecificationAPI, status_code=http_client.CREATED)
-    def post(self, spec, execution_id):
+    def post(self, spec, execution_id, no_merge=None):
```

Comment:
Minor thing - you can also default this argument to `False` and then specify `arg_types`  argument in `jsexpose` and it will correctly handle and cast the query param.